rustfmt.toml

```diff
@@ -3,20 +3,19 @@ array_width = 80
 attr_fn_like_width = 80
 chain_width = 80
 comment_width = 100
-#condense_wildcard_suffixes = true
-#enum_discrim_align_threshold = 40
+condense_wildcard_suffixes = true
 fn_call_width = 80
-#fn_single_line = true
-#format_code_in_doc_comments = true
-#format_macro_matchers = true
-#format_strings = true
-#group_imports = "StdExternalCrate"
-#hex_literal_case = "Lower"
-#imports_granularity = "Crate"
+format_code_in_doc_comments = true
+format_macro_matchers = true
+group_imports = "StdExternalCrate"
+hex_literal_case = "Upper"
```

Comment:
This feels like an odd choice - lower-case hex literals are easier to read/compare because each one has its own "texture" if you will, due to the transition between digits and characters.

I guess I don't have a compelling reason for `"Lower"` here (and I guess I must've missed that we set it up to "Upper" in other repos, aside from the compiler which uses "Lower") - but I am curious how we landed on a different value than what was previously used in this repo, considering that it is by far our largest project.

Reply:
We actually don't enforce it in other repos - but I've noticed here that sometimes we use upper case and sometimes lower case and I think it is probably a good idea to have consistency. I'm fine with switching to always using lowercase if you think that's better.

Reply:
If you have a preference, that's fine by me. I have a tendency to get what I call "hex blindness" with anything but the shortest upper case strings of hex digits, which is why I use lower-case hexadecimal by default, but I can certainly cope with whatever we end up with here. Most of what I'm working on is predominately in the compiler anyway, so it isn't a big deal at all.

Reply:
Changed `hex_literal_case` to `Lower`.